casr/src/bin/casr-libfuzzer.rs

```diff
@@ -40,7 +37,7 @@ fn main() -> Result<()> {
                 .action(ArgAction::Set)
                 .value_name("SECONDS")
                 .help("Timeout (in seconds) for target execution [default: disabled]")
-                .value_parser(clap::value_parser!(u64).range(1..))
+                .value_parser(clap::value_parser!(u64).range(0..))
```

Comment:
can we add default value = 0 and document that it means disabling timeout?

Reply:
done

Reply:
I mean that 0 value actually means disabling timeout, and set default value to zero

Reply:
like here https://www.llvm.org/docs/LibFuzzer.html#options

Reply:
I changed the description